sticker/src/tensorflow/tagger.rs

```diff
@@ -150,8 +150,8 @@ impl TaggerGraph {
         let logdir_op = Self::add_op(&graph, op_names::LOGDIR_OP)?;
```

Comment:
If I'm not mistaken, neither, `logdir_op`, `graph_write_op` nor `summary_init_op` should be present in 0.4 graphs.

Reply:
Thanks! Indeed, this should be an `Option` as well.

Reply:
But it is indeed surprising that it loaded the graph at all.